templates/UmbracoProject/appsettings.Production.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "./appsettings-schema.json",
+  "Umbraco": {
+    "CMS": {
+      "Runtime": {
+        "Mode": "Production"
+      },
+      "Global": {
+        "UseHttps": true
+      },
+      "ModelsBuilder": {
+        "ModelsMode": "Nothing"
+      },
+      "WebRouting": {
+        "UmbracoApplicationUrl": "https://localhost:HTTPS_PORT_FROM_TEMPLATE"
```

Comment:
Adding this might be a bit tricky, as leaving the `localhost` URL will break the keep-alive task and generated URLs in notifications (like password reset, invites, health check reports, etc.).

Maybe we should revisit validating this setting alltogether?

Reply:
The `UmbracoApplicationUrl` is not set as environment variable in the `launchSettings.json` file, so it's correctly configured when testing from source (this file also contains the actual URLs, so it makes more sense as well).